crates/config/src/lib.rs

```diff
@@ -957,12 +957,7 @@ impl Config {
             return match rx.recv_timeout(Duration::from_secs(1)) {
                 Ok(res) => res,
                 Err(RecvTimeoutError::Timeout) => {
-                    eprintln!(
-                        "{}",
-                        yansi::Paint::yellow(
-                            "Pulling Docker image for eof-solc, this might take some time..."
-                        )
-                    );
+                    // sh_warn!("Pulling Docker image for eof-solc, this might take some time...");
```

Comment:
any reason to comment this message, or just a leftover?

Reply:
Ah.. had missed re-enabling this, there is a circular dependency here that has to be fixed.

Reply:
Fixed this by allowing `eprintln!` to occur here given it is outside of a common execution path and there is no simple fix for the circular dependency without a major refactor